apps/examples/webserver_test/Makefile

```diff
@@ -0,0 +1,160 @@
+###########################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
+# apps/examples/webserver_test/Makefile
+#
+#   Copyright (C) 2007-2008, 2010-2012 Gregory Nutt. All rights reserved.
```

Comment:
nuttx license is not required if it's new file. So please remove license.

Reply:
done